scripts/tier_analyzer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+import argparse
+import re
+import subprocess
+import sys
+
+
+def check_virtualenv():
+    """Check if we are in virtualenv and if not, exit."""
+    if not hasattr(sys, 'real_prefix'):
+        print('You must activate CFME virtualenv in oder to run this script.')
+        sys.exit(1)
```

Comment:
**OPTIONAL:** Instead of exiting consider throwing an exception (which uncaught will exit but with more diagnostic information).   See scripts/coverage_report_jenkins.py as an example.